axonserver-cli/src/main/java/io/axoniq/cli/CommandOptions.java

```diff
@@ -19,10 +19,14 @@
     public static final Option ADDRESS = Option.builder("S").hasArg().longOpt("server").desc("Server to send command to (default http://localhost:8024)")
                                                .build();
     public static final Option APPLICATION = Option.builder("a").hasArg().longOpt("application").required().desc("Name of the application").build();
-    public static final Option ROLES = Option.builder("r").required().hasArgs().valueSeparator(',').longOpt("roles").desc("Roles for the application (READ, WRITE, ADMIN), use role@context for other context than default").build();
-    public static final Option USER_ROLES = Option.builder("r").hasArgs().valueSeparator(',').longOpt("roles").desc("[Optional] roles for the user (READ, ADMIN)").build();
+    public static final Option ROLES = Option.builder("r").required().hasArgs().valueSeparator(',').longOpt("roles")
+                                             .desc("Roles for the application, use role@context").build();
```

Comment:
can be useful in the description to have the list of all possible roles for applications